### PR TITLE
adding warning for mtu ignoring

### PR DESCRIPTION
**Pull request by ss1h2a3tw**
Because of unprivileged container cannot set MTU by configuration,
Adding a warning can be better.
Closing https://github.com/lxc/lxc/issues/1602
Signed-off-by: Shane Chen <ss1ha3tw@gmail.com>

### CODE REVIEW

File: src/lxc/conf.c [hunk 1]
```diff
@@ -3351,6 +3351,8 @@ int lxc_assign_network(const char *lxcpath, char *lxcname,
 		netdev = iterator->elem;
 
 		if (netdev->type == LXC_NET_VETH && !am_root) {
+			if (netdev->mtu)
+				WARN("ignoring MTU due to unprivileged");
```
Comment (brauner) [inline src/lxc/conf.c]: Could you make this an `INFO()` and reword it to: "mtu ignored due to insufficient privilege"? That would be awesome.

Comment (ss1h2a3tw) [inline src/lxc/conf.c]: OK

**Comment by brauner** [comment]
jenkins: test this please
